builder/builder.go

```diff
@@ -464,7 +464,7 @@ func (h *HeighlinerBuilder) queueTmpDirRemoval(tmpDir string, start bool) {
 // registerSigIntHandler will delete tmp dirs on ctrl+c
 func (h *HeighlinerBuilder) registerSigIntHandler() {
 	c := make(chan os.Signal)
-	//nolint:govet
+	//nolint:govet,staticcheck
 	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
```

Comment:
Instead of ignoring the lint, how using staticcheck's suggestion? In this case, it's best practice. 

```
c :- make(chan os.Signal, 1)
```

☝️ will fix the issue

Reply:
We should also delete the `//nolint:govet`. Giving the channel a buffer passes vet. Fyi, ignoring `go vet` is almost always a code smell.

Reply:
I tried this in the past, but it didn't work. Did not catch Ctrl+C

Reply:
Seems to be working :man_shrugging: . Thanks!